backend/src/main/resources/liquibase/create-tables-changelog-1.0.xml

```diff
@@ -1238,4 +1238,86 @@
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="connection_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_versioning_blob_entity">
+        <createTable tableName="lineage_versioning_blob">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="repository_id" type="int8">
+                <constraints nullable="true"/>
+            </column>
+            <column name="blob_sha" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="blob_type" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_experiment_run_entity">
+        <createTable tableName="lineage_experiment_run">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="experiment_run_id" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="indexes-on-lineage" author="lezhevg">
+        <createIndex indexName="lineage_versioning_blob_index"
+          tableName="lineage_versioning_blob" unique="false">
+            <column name="repository_id" type="int8">
+            </column>
+            <column name="blob_sha" type="varchar(256)">
+            </column>
+            <column name="blob_type" type="varchar(256)">
+            </column>
+        </createIndex>
+        <createIndex indexName="lineage_experiment_run_index"
+          tableName="lineage_experiment_run" unique="false">
+            <column name="experiment_run_id" type="varchar(256)"/>
+        </createIndex>
+        <createIndex indexName="lineage_connection_index"
+          tableName="lineage_connection" unique="false">
+            <column name="element_id" type="int8">
+            </column>
+            <column name="connection_type" type="int4">
+            </column>
+        </createIndex>
+        <createIndex indexName="lineage_connection_index2"
+          tableName="lineage_connection" unique="false">
+            <column name="entity_id" type="int8">
+            </column>
+            <column name="connection_type" type="int4">
+            </column>
+            <column name="entity_type" type="int4">
+            </column>
+        </createIndex>
+    </changeSet>
```

Comment:
I think these need to be in a different changeset. @ravishetye ?

Reply:
yes different changeset.  and each changeset with a precondition similar to https://github.com/VertaAI/modeldb/blob/master/backend/src/main/resources/liquibase/create-index-changelog.xml#L62-L65

Reply:
fixed